mongodb_migrations/config.py

```diff
@@ -37,10 +37,10 @@ def _from_console(self):
                                      help="port of MongoDB")
         self.arg_parser.add_argument('--database', metavar='d',
                                      help="database of MongoDB", default=self.mongo_database)        
-        self.arg_parser.add_argument('--authuser', metavar='au',
-                                     help="username for auth database of MongoDB", default=self.mongo_auth_user)
-        self.arg_parser.add_argument('--authpw', metavar='p',
-                                     help="password for auth database of MongoDB", default=self.mongo_auth_password)
+        self.arg_parser.add_argument('--username', metavar='U',
+                                     help="username for auth database of MongoDB", default=self.mongo_username)
+        self.arg_parser.add_argument('--password', metavar='p',
```

Comment:
Sorry I didn't notice this before. We already '-p' for `port`, so we should use '-P' for `password`. Sorry about this

Reply:
done & pushed :)